e2e/backend.e2e.spec.ts

```diff
@@ -99,8 +99,13 @@ test.describe.serial('KubernetesBackend', () => {
         headers: { Authorization: `basic ${ auth }` },
         method:  'PUT',
       });
+      const text = await result.text();
```

Comment:
I gather this is about the occasional error message complaining about an invalid json packet. Shouldn't we replace all calls to `result.json()` with this sequence?

Reply:
Probably, but I didn't want to make this PR even bigger. We can do a follow-up for this.